Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/ProjectionIntegrityViolationDetection/AllNodesAreConnectedToARootNodePerSubgraph.feature

```diff
@@ -52,7 +52,8 @@ Feature: Run projection integrity violation detection regarding root connection
       | parentNodeAggregateId       | "sir-david-nodenborough"                  |
       | nodeName                    | "child-document"                          |
       | nodeAggregateClassification | "regular"                                 |
-    And the event NodeAggregateWasMoved was published with payload:
+    # we must apply the event directly as the state is so corrupt that catchup hooks might fail like the content cache flusher
+    And the event NodeAggregateWasMoved is hacky directly applied with payload:
```

Comment:
this is just a random idea in the totally optional last commit that we can drop as well. The idea is that we dont even really publish the event but call `DoctrineDbalContentGraphProjection::apply` which is of course super hacky and i know but it will make sure no catchup hooks are triggered.

The alternative is to create the state by sql by hand and thats a mess probably?

Reply:
Moved the test to the doctrine dbal and using now raw sql to create the invalid state.